hydra_server/server/complexmodels.py

```diff
@@ -1480,6 +1480,7 @@ class Project(Resource):
         ('status',      Unicode(default='A', pattern="[AX]")),
         ('cr_date',     Unicode(default=None)),
         ('created_by',  Integer(default=None)),
+        ('appdata',     AnyDict(min_occurs=0, max_occurs=1, default=None)),
```

Comment:
Did you prefer anydict over json to be more flexible?

This field should also appear into the doc.

Reply:
This is just for consistency with other paramters in the file

Reply:
Added to docstring